modules/setting/setting.go

```diff
@@ -1073,6 +1084,44 @@ func NewContext() {
 	UI.ShowUserEmail = Cfg.Section("ui").Key("SHOW_USER_EMAIL").MustBool(true)
 
 	HasRobotsTxt = com.IsFile(path.Join(CustomPath, "robots.txt"))
+
+	extensionReg := regexp.MustCompile(`\.\w`)
+	for _, sec := range Cfg.Section("markup").ChildSections() {
+		name := strings.TrimLeft(sec.Name(), "markup.")
+		if name == "" {
+			log.Warn(sec.Name() + " name is empty, ignored")
+			continue
+		}
+
+		extensions := sec.Key("FILE_EXTENSIONS").Strings(",")
+		var exts = make([]string, 0, len(extensions))
+		for _, extension := range extensions {
+			if !extensionReg.MatchString(extension) {
+				log.Warn(sec.Name() + " FILE_EXTENSIONS " + extension + " is invalid, ignored")
```

Comment:
nit: add info of what is ignored, this way it looks like only option is ignored, not whole parser

Reply:
Yes, I think maybe someone write a wrong extension so we could ignore only that extension.

Reply:
Can you change it to something like:
`sec.Name() + ' file extension "' + extension + '" is invalid. Extension ignored"`

Also other lines (I wrote comment to wrong one) where you use `continue` can be more descriptive as you are ignoring whole parser (on this line you ignore only extension, but on other lines you ignore everything - same text, different meaning).

Reply:
@Morlinest done